lms/services/lti.py

```diff
@@ -0,0 +1,86 @@
+"""
+LTI service.
+
+This is a place to put LTI-related methods where they can be called from
+anywhere in the code, they can have access to the request and context, and they
+can have state (e.g. to cache the results of computations).
+"""
+import pylti.common
+
+from lms.services import LTILaunchVerificationError
+from lms.services import NoConsumerKey
+from lms.services import ConsumerKeyError
+from lms.services import LTIOAuthError
+
+
+class LTIService:
+    """A collection of LTI-related methods."""
```

Comment:
The definition of the service's responsibility is a bit vague. Do you have an idea of what other functionality might fit here in future?

Reply:
Nope. It may be that this only ever contains launch verification. For now I think I'll rename it to `LaunchVerifier`